Testing/RegressionTesting/configfiles/test-medium-long.xml

```diff
@@ -117,6 +114,7 @@
 
     <RecorderParams class="XmlGrowthRecorder" name="RecorderParams">
       <probedNListFileName name="prbNListFileName" type="InputFile">../configfiles/NList/probedNList_900.xml</probedNListFileName>
+      <resultFileName name="resultFileName">../Testing/RegressionTesting/TestOutput/test-medium-long-out.xml</resultFileName>
     </RecorderParams>
   </ModelParams>
 </BGSimParams>
```

Comment:
Nit-pick: no newline here (and it seems some other files) ad indicated by the red circle symbol. Not sure if it matters, though I figured I would point it out.